next/components/forms/segments/AccountSections/IntroSection/IntroSection.tsx

```diff
@@ -19,6 +20,13 @@ const IntroSection = () => {
         text={t('account_section_intro.header_text')}
       />
       <div className="w-full max-w-screen-1.5lg m-auto py-6 lg:py-16">
+        <ActualBlock
+          title="Digitálna platba dane z nehnuteľností "
+          text="Daň z nehnuteľností pre rok 2023 spracovávame od 20. apríla do 30. mája. Keď bude vaša daň z nehnuteľností pripravená, pošleme vám informačný e-mail a nájdete ju v konte, v časti Dane a poplatky."
```

Comment:
I understand the sentiment of keeping this inline, but since it's quite possible this won't be pulled from any cms anytime soon - let's just keep it together with the rest of the 'account' translations (again, feel free to copy the same text into english version)